docker-compose.yml

```diff
@@ -0,0 +1,84 @@
+version: "3.1"
+
+networks:
+  net:
+
+services:
+  worker:
+    image: woy/worker1
+    #image: ghcr.io/hawks-atlanta/worker-java:latest
+    container_name: worker
+    restart: on-failure
+    depends_on:
+      - metadata
+    ports:
+      - "127.0.0.1:1099:1099" # RMI
+    environment:
+      METADATA_BASEURL: "http://metadata:8080/api/v1"
+      VOLUME_BASE_PATH: "/var/capy/store"
+      AVAILABLE_VOLUMES: "1,2,3"
+    networks:
+      - net
+
+  metadata:
+    image: ghcr.io/hawks-atlanta/metadata-scala:latest
+    container_name: metadata
+    restart: on-failure
+    depends_on:
+      - postgres-db
+    ports:
+      - "127.0.0.1:8082:8080"
+    environment:
+      DATABASE_HOST: postgres-db
+      DATABASE_PORT: 5432
+      DATABASE_NAME: metadatadb
+      DATABASE_USER: username
+      DATABASE_PASSWORD: password
+    networks:
+      - net
+
+  authentication:
+    image: ghcr.io/hawks-atlanta/authentication-go:latest
+    container_name: authentication
+    restart: on-failure
+    depends_on:
+      - postgres-db
+    ports:
+      - "127.0.0.1:8081:8080"
+    environment:
+      DATABASE_ENGINE: postgres
+      DATABASE_DSN: "host=postgres-db user=username password=password dbname=authdb port=5432 sslmode=disable"
+    networks:
+      - net
+
+  postgres-db:
+    image: postgres:latest
+    container_name: postgres-db
+    restart: on-failure
+    ports:
+      - "127.0.0.1:5432:5432"
+    environment:
+      POSTGRES_MULTIPLE_DATABASES: authdb,metadatadb
+      POSTGRES_PASSWORD: password
+      POSTGRES_USER: username
+    volumes:
+      - ./.docker-psql-mult-db:/docker-entrypoint-initdb.d/init.sh
+      - ../docker-mounts/capydb:/var/lib/postgresql/data
```

Comment:
Do not mount the volume on which database will be stored, we have no need to maintain persistence